projects/packages/videopress/src/client/lib/get-media-token/index.ts

```diff
@@ -1,28 +1,40 @@
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
 /**
  * Internal dependencies
  */
 import { VideoGUID } from '../../block-editor/blocks/video/types';
-import {
+import { MEDIA_TOKEN_SCOPES } from './types';
+/**
+ * Types
+ */
+import type {
 	MediaTokenScopeProps,
 	MediaTokenScopeAdminAjaxResponseBodyProps,
 	MediaTokenProps,
-	MEDIA_TOKEN_SCOPES,
 	AdminAjaxTokenProps,
 	GetMediaTokenArgsProps,
 } from './types';
 
+const debug = debugFactory( 'videopress:get-media-token' );
+
+// Lifetime of the token in milliseconds.
+const TOKEN_LIFETIME = 1000 * 60 * 60 * 12; // 12 hours
+
 /**
- * Return media token data hiting the admin-ajax endpoint.
+ * Request media token data hiting the admin-ajax endpoint.
  *
  * @param {MediaTokenScopeProps} scope  - The scope of the token to request.
  * @param {GetMediaTokenArgsProps} args - function arguments.
  * @returns {MediaTokenProps}             Media token data.
  */
-const getMediaToken = function (
+const requestMediaToken = function (
 	scope: MediaTokenScopeProps,
 	args: GetMediaTokenArgsProps = {}
 ): Promise< MediaTokenProps > {
-	const { id, guid, adminAjaxAPI: adminAjaxAPIArgument, filename } = args;
+	const { id = 0, guid, adminAjaxAPI: adminAjaxAPIArgument, filename } = args;
```

Comment:
what's the logic behind this change?

Reply:
Nice question. Something that we've talked about with @lhkowalski.
Passing the post ID when requesting the token is a mechanism to handle access to the private video, depending on the video embedded. 

It's `0` as default because, in the editor context, we don't want to limit the privacy scope.

PR https://github.com/Automattic/jetpack/pull/23949

Reply:
Exactly. Here is the trace of the field on the backend, on `trunk`:

- where we read it: https://github.com/Automattic/jetpack/blob/trunk/projects/packages/videopress/src/class-ajax.php#L84
- where it's used, the default value and the meaning of the field: https://github.com/Automattic/jetpack/blob/trunk/projects/packages/videopress/src/class-ajax.php#L121